helm/panda/charts/jedi/sandbox/init-jedi

```diff
@@ -2,15 +2,14 @@
 /opt/panda/bin/panda_common-install_igtf_ca > /var/log/panda/install_igtf_ca.log 2>&1
 
 # fetch info from CRIC before running services
-tmpExe=`mktemp`
-cat <<EOT >> ${tmpExe}
-/opt/panda/usr/bin/panda_server-schedconfig_json
-/opt/panda/usr/bin/panda_server-configurator
-EOT
-chmod +x ${tmpExe}
-
 if [[ -z "${PANDA_NO_ROOT}" ]]; then
-  runuser -u atlpan -g zp -- ${tmpExe}
+  runuser -u atlpan -g zp -- /opt/panda/usr/bin/panda_server-schedconfig_json
```

Comment:
Why do you need this change? The downside is that you have to specify the same string twice

if ...
   ...
  runuser -u atlpan -g zp -- \<string\>
else
  ...
  \<string\>
fi


every time you add a new command.

Reply:
I updated it back. However I changed the mktemp to a fixed filename.
Since during the debugging at the beginning, I cleaned some tmp files and it caused some issues. I now put them in /data/panda directory.